--- origsrc/lv2-1.6.0/plugins/eg01-amp.lv2/wscript	2013-02-10 18:19:09.000000000 -0600
+++ src/lv2-1.6.0/plugins/eg01-amp.lv2/wscript	2013-09-15 21:41:45.213064300 -0500
@@ -32,7 +32,7 @@ def build(bld):
     bundle = 'eg-amp.lv2'
 
     # Make a pattern for shared objects without the 'lib' prefix
-    module_pat = re.sub('^lib', '', bld.env.cshlib_PATTERN)
+    module_pat = re.sub('^(lib|cyg)', '', bld.env.cshlib_PATTERN)
     module_ext = module_pat[module_pat.rfind('.'):]
 
     # Build manifest.ttl by substitution (for portable lib extension)
--- origsrc/lv2-1.6.0/plugins/eg02-midigate.lv2/wscript	2013-02-10 18:19:06.000000000 -0600
+++ src/lv2-1.6.0/plugins/eg02-midigate.lv2/wscript	2013-09-15 21:42:10.309499700 -0500
@@ -30,7 +30,7 @@ def build(bld):
     bundle = 'eg-midigate.lv2'
 
     # Make a pattern for shared objects without the 'lib' prefix
-    module_pat = re.sub('^lib', '', bld.env.cshlib_PATTERN)
+    module_pat = re.sub('^(lib|cyg)', '', bld.env.cshlib_PATTERN)
     module_ext = module_pat[module_pat.rfind('.'):]
 
     # Build manifest.ttl by substitution (for portable lib extension)
--- origsrc/lv2-1.6.0/plugins/eg03-metro.lv2/wscript	2013-02-10 18:19:09.000000000 -0600
+++ src/lv2-1.6.0/plugins/eg03-metro.lv2/wscript	2013-09-15 21:42:18.860988900 -0500
@@ -30,7 +30,7 @@ def build(bld):
     bundle = 'eg-metro.lv2'
 
     # Make a pattern for shared objects without the 'lib' prefix
-    module_pat = re.sub('^lib', '', bld.env.cshlib_PATTERN)
+    module_pat = re.sub('^(lib|cyg)', '', bld.env.cshlib_PATTERN)
     module_ext = module_pat[module_pat.rfind('.'):]
 
     # Build manifest.ttl by substitution (for portable lib extension)
--- origsrc/lv2-1.6.0/plugins/eg04-sampler.lv2/wscript	2013-02-10 18:19:08.000000000 -0600
+++ src/lv2-1.6.0/plugins/eg04-sampler.lv2/wscript	2013-09-15 21:42:24.329301600 -0500
@@ -35,7 +35,7 @@ def build(bld):
     bundle = 'eg-sampler.lv2'
 
     # Make a pattern for shared objects without the 'lib' prefix
-    module_pat = re.sub('^lib', '', bld.env.cshlib_PATTERN)
+    module_pat = re.sub('^(lib|cyg)', '', bld.env.cshlib_PATTERN)
     module_ext = module_pat[module_pat.rfind('.'):]
 
     # Build manifest.ttl by substitution (for portable lib extension)
